.github/workflows/causal_lm_cpp.yml

```
@@ -395,4 +395,47 @@ jobs:
                   raise RuntimeError(f'Missing "{ref=}" from predictions')
               predictions = predictions[:idx] + predictions[idx + len(ref):]
           "
+          echo Phi-1_5 passed
+          
+  cpp-greedy_causal_lm-redpajama-3b-chat:
+    runs-on: ubuntu-20.04-8-cores
```

```suggestion
    runs-on: ubuntu-20.04
```
Can you try the smaller runner?